README.md

```diff
@@ -43,6 +43,12 @@ If you want to specify the output file, add the `--output` option:
 npm audit --json | npm-audit-html --output report.html
 ```
 
+You can also fully customize the generated report by providing `--template` option followed by your own handlebars template: 
```

Comment:
It might be helpful to add a link to the existing Handlebars template for an example.